frontend/src/locales/es/translation.json

```diff
@@ -168,6 +168,7 @@
     "ChartTypeError": "Seleccione un tipo de gráfico",
     "DisplayHorizontalScroll": "Mostrar con desplazamiento horizontal",
     "ShowDataLabels": "Mostrar etiquetas de datos",
+    "ComputePercentages": "!!NEED TRANSLATION!!",
```

Comment:
We prefer not to change the es and pt translation files, only the English one, that way is easier for us later.

Reply:
Ok